src/base/init/init.c

```diff
@@ -349,7 +349,7 @@ void low_mem_init(void)
 
   mem_base = mem_reserve(memsize, dpmi_size);
   if (config.cpu_vm == CPUVM_KVM || config.cpu_vm_dpmi == CPUVM_KVM) {
-    init_kvm_monitor();
+    init_kvm_monitor(memsize + dpmi_size);
```

Comment:
Your log says "behind main pool",
but that looks after main pool?

Reply:
yes, corrected